kasa/smart/modules/fanmodule.py

```diff
@@ -55,10 +56,14 @@ def fan_speed_level(self) -> int:
         return self.data["fan_speed_level"]
 
     async def set_fan_speed_level(self, level: int):
-        """Set fan speed level."""
-        if level < 1 or level > 4:
-            raise ValueError("Invalid level, should be in range 1-4.")
-        return await self.call("set_device_info", {"fan_speed_level": level})
+        """Set fan speed level, 0 for off, 1-4 for on."""
+        if level < 0 or level > 4:
+            raise KasaException("Invalid level, should be in range 0-4.")
```

Comment:
We should use standard exceptions wherever it makes sense, so use `ValueError` here.

Reply:
Done